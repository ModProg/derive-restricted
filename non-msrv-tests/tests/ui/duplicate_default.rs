#[derive(derive_where::DeriveWhere)]
#[derive_where(Default; T)]
enum Test<T> {
	#[derive_where(default)]
	A(T),
	#[derive_where(default)]
	B(T),
}

fn main() {}
