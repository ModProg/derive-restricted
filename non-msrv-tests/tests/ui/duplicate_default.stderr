error: multiple `default` options in enum
 --> tests/ui/duplicate_default.rs:6:17
  |
6 |     #[derive_where(default)]
  |                    ^^^^^^^
